server/src/main/java/org/cloudfoundry/identity/uaa/scim/jdbc/JdbcScimGroupProvisioning.java

```diff
@@ -74,6 +78,7 @@ public JdbcScimGroupProvisioning(
         super(jdbcTemplate, pagingListFactory, new ScimGroupRowMapper());
 
         this.jdbcTemplate = jdbcTemplate;
+        jdbcIdentityZoneProvisioning = new JdbcIdentityZoneProvisioning(jdbcTemplate); // TODO inject
```

Comment:
please enhance
https://github.com/cloudfoundry/uaa/blob/develop/uaa/src/main/webapp/WEB-INF/spring/scim-endpoints.xml#L112
with jdbcIdentityZoneProvisioning

Reply:
no bean jdbcIdentityZoneProvisioning found for ref